strawberry/aiohttp/views.py

```diff
@@ -307,8 +308,10 @@ async def parse_multipart_body(self, request: web.Request) -> dict:
             raise web.HTTPBadRequest(reason="File(s) missing in form data")
 
     def render_graphiql(self) -> web.StreamResponse:
-        html_string = self.graphiql_html_file_path.read_text()
-        html_string = html_string.replace("{{ SUBSCRIPTION_ENABLED }}", "true")
+        html_string = get_graphiql_html(
+            subscription_enabled=True,
+            graphiql_html_file_path=self.graphiql_html_file_path,
```

Comment:
aiohttp can use the same default graphiql file as the other integrations. The `HTTPHandler.graphiql_html_file_path` property can then be removed and the `graphiql_html_file_path` argument as well :)